tls.go

```diff
@@ -232,31 +233,29 @@ type TLSLog struct {
 }
 
 func (z *TLSConnection) GetLog() *TLSLog {
-	handshake := z.Conn.GetHandshakeLog()
-	if !z.flags.KeepClientLogs {
-		handshake.ClientHello = nil
-		handshake.ClientKeyExchange = nil
-		handshake.ClientFinished = nil
-	}
-	var heartbleed *tls.Heartbleed
-	if z.flags.Heartbleed {
-		heartbleed = z.Conn.GetHeartbleedLog()
-	} else {
-		heartbleed = nil
-	}
-	return &TLSLog{
-		HandshakeLog:  handshake,
-		HeartbleedLog: heartbleed,
+	if z.log == nil {
```

Comment:
Previously, callers had to be careful to call GetLog() as late as possible, which was awkward (and unnecessary).